src/ilcdlib/entity/lcia.py

```diff
@@ -57,10 +58,38 @@ def get_impact_set(self, scenario_names: dict[str, str]) -> ImpactSet:
                 scenario_names=scenario_names,
                 scope_to_units_mapper=self.scope_to_units_mapper,
             )
+
+        self._extract_and_set_a1a2a3_impact(impacts)
+
         if len(ext) == 0:
             del impacts["ext"]
         return ImpactSet(**impacts)  # type: ignore
 
+    @staticmethod
+    def __process_a1a2a3_impact(scope_set: ScopeSet) -> None:
+        if scope_set.A1A2A3 is None and (scope_set.A1 or scope_set.A2 or scope_set.A3):
+            s: float = 0
+            unit = None
+
+            for A in (scope_set.A1, scope_set.A2, scope_set.A3):
```

Comment:
uppercased local var might trigger linter warnings as it breaks convention. Could you rename pls